data/scenariolens-avd.data.json

```
@@ -0,0 +1,184 @@
+    "recommendation": {
+      "title": "Review your central profile store architecture and incorporate FSLogix if you are using multi-session",
+      "context": "",
+      "links": []
+    },
+    "children": [
+      {
+        "title": "(Storage) Storage requirements should be optimized for performance in the Azure Virtual Desktop service",
+        "context": "[Storage sizing for Azure Files and Azure NetApp Files](https://docs.microsoft.com/azure/architecture/example-scenario/wvd/windows-virtual-desktop-fslogix#performance-requirements) needs to align with IOPS Requirements",
+        "id": "10fd7357-e5d3-4a22-b5e8-5fe698651ab6"
+      },
+      {
+        "title": "(Storage) Size of the volumes should reflect the performance needed",
+        "context": "Consideration of the size of the volume where user profiles will be stored on a per user profile size basis",
+        "id": "a46ddf00-2fdf-4e61-a7bd-50be19bcd97e"
+      },
+      {
+        "title": "(Storage - Azure Files) SKU of the Azure Files Storage Account",
+        "context": "Light users - Standard File Shares, Medium users - Standard/Premium File Shares, Heavy users - Premium File Shares, Power users - Premium File Shares",
+        "id": "2eb28737-ebc8-4e46-84e5-3ee61187e42f"
+      },
+      {
+        "title": "The chosen storage service for FSLogix",
+        "context": "Azure Files ([Azure AD Domain Services](https://docs.microsoft.com/azure/virtual-desktop/create-profile-container-adds) or [Active Directory Domain Services](https://docs.microsoft.com/azure/virtual-desktop/create-file-share), [Azure NetApp Files](https://docs.microsoft.com/azure/virtual-desktop/create-fslogix-profile-container), [VM-based file share](https://docs.microsoft.com/azure/virtual-desktop/create-host-pools-user-profile))",
+        "id": "196806be-9bba-4c1a-840f-21295381d2b5"
+      },
+      {
+        "title": "[FSLogix Cloud Cache](https://docs.microsoft.com/fslogix/cloud-cache-resiliency-availability-cncpt) for Business Continuity & Disaster Recovery",
+        "context": "FSLogix Cloud Cache is used for best practice failover and quick recovery scenarios",
+        "id": "ff539c70-e892-46a8-ab7c-9a65a12dc411"
+      },
+      {
+        "title": "Backup solution for the chosen storage service that FSLogix will be using",
+        "context": "It is just as vital to implement a backup solution next to your disaster recovery solution. [Azure NetApps Files supports scheduled snapshots](https://docs.microsoft.com/azure/azure-netapp-files/azure-netapp-files-manage-snapshots) and Azure Backup should be considered for [File Shares](https://docs.microsoft.com/azure/backup/azure-file-share-backup-overview)",
+        "id": "caaadea9-d365-4b31-bf37-02b0db301fa3"
+      },
+      {
+        "title": "Design your storage to reside as close as possible to your session hosts",
+        "context": "This increases performance and latency for a better end user experience",
```

This increases latency for better user experience? :D

```suggestion
        "context": "This increases performance and improves latency for a better end user experience.",
```